docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -0,0 +1,341 @@
+# ClusterClass
+
+The ClusterClass feature introduces a new way to create clusters which reduces boilerplate and enables flexible and powerful customization of clusters.
+ClusterClass is a powerful abstraction implemented on top of existing interfaces and offering a set of tools and operations to streamline cluster lifecycle management while maintaining the same underlying API.
+
+This tutorial covers using Cluster API and ClusterClass to create a Kubernetes cluster and how to perform a one-touch Kubernetes version upgrade.
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+This guide covers creating a ClusterClass cluster with the CAPD provider.
+</aside>
+
+## Installation
+
+### Prerequisites
+
+#### Install tools
+
+This guide requires the following tools are installed: 
+- Install and setup [kubectl] and [clusterctl]
+- Install [Kind] and [Docker]
+
+For more information see the [quick start guide].
```

Comment:
Up to you but I would probably drop this sentence. As a reader I would be confused at which part of the Quickstart to look at for more information as I don't know which part also applies to ClusterClass and which doesn't.

Are we referencing to a specific section of the Quickstart? (then we could maybe make the link more specific)

Reply:
I think you're right - better to make this guide stand by itself.